lib/spark_api/models/subresource.rb

```diff
@@ -16,7 +16,7 @@ def find_by_id(id, parent_id, arguments={})
 
       def parse_date_start_and_end_times(attributes)
         # Transform the date strings
-        unless attributes['Date'].nil?
+        unless attributes['Date'].nil? && attributes['Date'].empty?
           date = Date.strptime attributes['Date'], '%m/%d/%Y'
           ['StartTime','EndTime'].each do |time|
             next if attributes[time].nil?
```

Comment:
I believe you'll want a similar empty check for `StartTime` and `EndTime` here. Based on @joshcom's API review earlier today, those fields have the same implication.

Reply:
Will do, thanks Wade!